build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/views/rbs/async_client_class.rb

```diff
@@ -0,0 +1,171 @@
+# frozen_string_literal: true
+
+module AwsSdkCodeGenerator
+  module Views
+    module RBS
+      class AsyncClientClass < View
```

Comment:
Not reviewing this in depth since you copied most of this - can you do a sanity test and try to validate the rbs is correct?